packages/nestjs-zod/src/__e2e_tests__/openapi.test.ts

```diff
@@ -13,6 +13,10 @@ import request from 'supertest';
 import { setupApp } from '../testUtils';
 import { ZodSerializerDto } from '../serializer';
 
+beforeEach(() => {
+    z.globalRegistry.clear()
+})
```

Comment:
If you run multiple tests at the same time (e.g. `pnpm test`) then all schemas are going into the global registry. This will throw an error if two tests use the same id for a schema.